### DISCUSSION TITLE
End-to-end example

### DISCUSSION THREAD

**Discussion by superpi15** [Xilinx/finn#454] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/6512963/143612696-f00ca907-345e-4bf0-81bf-53f552cc11d0.png)
Hi friends, I'm running end to end example provided by FINN.
I encounter a problem as shown in the picture.
It is at the "5. Deployment and Remote Execution" step. 
After we successfully access Pynq-Z2, we are deploying. 
May I inquire your suggestions? Any hint is welcome. Thank you very much.

**Comment by rpitonak** [comment, marked as answer]
Hi @superpi15.

Please, see my answer in another discussion.

https://github.com/Xilinx/finn/discussions/442#discussioncomment-1675720

This should resolve your issue. Let me know if you need further help to make the example work.

**Reply by superpi15** [reply]
Thank you very much for the solution. 
I've spent much efforts but still cannot resolve this until now.
With your solution my Pynq-Z2 works well now.
